app/src/androidTest/java/org/maplibre/android/plugins/annotation/CircleManagerTest.java

```diff
@@ -1,23 +1,24 @@
 // This file is generated.
 
-package com.mapbox.mapboxsdk.plugins.annotation;
+package org.maplibre.android.plugins.annotation;
 
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-
-import com.mapbox.mapboxsdk.geometry.LatLng;
-import com.mapbox.mapboxsdk.plugins.testapp.activity.TestActivity;
-import com.mapbox.mapboxsdk.plugins.BaseActivityTest;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.maplibre.android.style.layers.Property.CIRCLE_PITCH_ALIGNMENT_MAP;
+import static org.maplibre.android.style.layers.Property.CIRCLE_PITCH_SCALE_MAP;
+import static org.maplibre.android.style.layers.Property.CIRCLE_TRANSLATE_ANCHOR_MAP;
 
-import timber.log.Timber;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.maplibre.android.plugins.BaseActivityTest;
+import org.maplibre.android.plugins.annotation.annotation.CircleManager;
```

Comment:
Hello, I read your PR because I am interested in SymbolManager and I realize that your package double annotation is normal?

Reply:
That's the way they (whoever built or worked on these. Plugins) have done it.. but in a previous (or upcoming) commit, I have fixed that by
- moving all the files one folder (or package) up
- removed the extra annotation folder
- and more importantly make sure that all import paths have been updated throughout the other.